blockchain/moralis/api v2/nft transfers lister/a490b7c6-e6f7-4bf1-8ef0-6124b1919f30.json

```diff
@@ -0,0 +1,256 @@
+{
+  "name": "blockchain/moralis/api v2/NFT transfers lister",
```

Comment:
We might want to have a "bounced" output too, like the other such (eg. io) nodes.